plugins/querytranslate/translate.go

```diff
@@ -71,6 +71,12 @@ func translateQuery(rsQuery RSQuery, userIP string) (string, error) {
 		}
 	}
 
+	// If no backend is passed for kNN, set it as `elasticsearch`
+	backendPassed := "elasticsearch"
```

Comment:
Backend property can be an enum with default value as `elasticsearch`.

Reply:
It's done!